install.sh

```
@@ -0,0 +1,371 @@
+#!/bin/bash
+
+set -e
+
+# Colors
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+YELLOW='\033[0;33m'
+BOLD='\033[1m'
```

Shellcheck (0.10.0)

[warning] BOLD appears unused. Verify use (or export if used externally).

(SC2034)